tests/saitests/py3/sai_qos_tests.py

```diff
@@ -335,6 +335,11 @@ def runTest(self):
         print("dst_port_id: %d, src_port_id: %d" %
               (dst_port_id, src_port_id), file=sys.stderr)
 
+        self.exec_cmd_on_dut(self.server, self.test_params['dut_username'],
+                             self.test_params['dut_password'],
+                             'ping {} -c 3'.format(dst_port_ip))
+        time.sleep(8)
```

Comment:
arp entry should have been populated by the ArpPopulate function?

Reply:
case testQosSaiSeparatedDscpQueueMapping was executed on t1 device.
test traffic is from T0 to T2, traffic's dst ip address 10.0.0.13 is an BGP neighbor, and relevant route is directly connect route, as below:
```
$ show ip bgp sum
... omitted ...
Neighbhor      V     AS    MsgRcvd    MsgSent    TblVer    InQ    OutQ  Up/Down      State/PfxRcd  NeighborName
-----------  ---  -----  ---------  ---------  --------  -----  ------  ---------  --------------  --------------
... omitted ...
10.0.0.13      4  65200       3236         71         0      0       0  00:46:50             6370  ARISTA07T2
... omitted ...

$ show ip route 10.0.0.13
Routing entry for 10.0.0.12/31\
  Known via \"connected\", distance 0, metric 0, best\  Last update 00:03:56 ago\
  * directly connected, PortChannel1011\
```

need to populate ARP of 10.0.0.13 on dut, when forward test traffic to dst 10.0.0.13.
Existing ArpPopulate function can not trigger DUT to learn correct dst mac of 10.0.0.13. so need to force DUT to launch "ping" to lear ARP.

Reply:
I looked at the ArpPopulate function and see that it has been written with the assumption of resolving addresses only on the T0-server links. I think the right place to put this change is the ArpPopulate function. If the dest ip is in the 10.0.0.0 network we should accordingly set the target address to that of the peer instead of the hardcoded 192.168.0.1

Reply:
resolved, in arppopulate() function, learn arp for all neighbor